src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/Calculator.cs

```diff
@@ -25,7 +25,7 @@ public static ResourceUtilization CalculateUtilization(in Snapshot first, in Sna
         long runtimeTickDelta = second.TotalTimeSinceStart.Ticks - first.TotalTimeSinceStart.Ticks;
 
         // Compute the total number of ticks available on the machine during that interval
-        double totalSystemTicks = runtimeTickDelta * systemResources.GuaranteedCpuUnits;
```

Comment:
Moved this systemResources.GuaranteedCpuUnits to ResourceUtilization class for consistency because systemResources.GuaranteedMemoryInBytes is used there too